src/simulation_presets.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * simulation preset structure
+ * {
+ *  populationSize: *number*, in N and > 9, such that sqrt(populationSize) in N,
+ *  hospitalEffectiveness: *number*, in [0, 1]
+ *  hygieneDisregard: *number*, in [0, 1] // default 1
+ *  hospitalCapacity: *number* , in [0, 1]
+ *  incubationPeriod: *number*, in [0, 50] and in N | default 6
+ *  infectionPeriod: *number*, in [2, 100] and in N | default 41
+ *  spreadRadius: *number*, in [2, sqrt(populationSize)] | default 1
+ *  testRate: *number*, in [0, 1] | default 0
+ *  testCooldown: *number* in [1, 100] and in N | default 7
+ *  quarantineType: *number* in [0, 1, 2],
+ *  quarantineEffectiveness: *number*, in [0, 1]
+ *  quarantinePercentage: (optional) *number*, in [0, 1]
+ *  quarantineDelay: *number*, in [0, 50] and in N,
+ *  quarantinePeriod: *number*, in [0, 50] and in N,
+ *  populationPreset, *number*, in [0, 1, 2]
+ * }
+ *
+ */
```

Comment:
It might be a good idea to have a simulation-presets/ folder and export each preset from a single file with its name.
Then you can have an index.js there that exports them all for convenience and you just have to 
```
import { GermanyPreset, SpainPreset } from "./simulationPresets"
```

Reply:
Thanks for the insight! I just added that folder. I exported the object of simulationPresets, as they would be exported together in index.js. Should I change them to individual exports, or add the individual preset exports?

Reply:
No, basically that's the job of the index.js, to group every object and export. Each object should be in a separate file identifying the preset, if we have many presets we can find them more easily this way

Reply:
Oh, I see! Just committed a new version, did you mean something like this?